src/v3/package.json

```diff
@@ -115,21 +115,31 @@
     "jest-junit": "^12.2.0",
     "lint-staged": "^12.2.1",
     "msw": "^1.1.1",
+    "nodemon": "^3.0.1",
     "postcss": "^8.4.13",
+    "postcss-loader": "^7.3.3",
+    "postcss-preset-env": "^8.4.1",
     "postcss-syntax": "^0.36.2",
     "preact-render-to-string": "^5.1.19",
     "preact-svg-loader": "^0.2.1",
     "properties": "^1.2.1",
+    "sass-loader": "^13.2.2",
     "sirv-cli": "^2.0.2",
+    "style-loader": "^3.3.3",
     "stylelint": "^14.16.1",
     "stylelint-use-logical-spec": "^5.0.0",
     "svgo": "^2.3.0",
+    "svg-inline-loader": "^0.8.2",
```

Comment:
Not sure, I copied this in so I can investigate `raw-loader`. Is there something specific about `svg-inline-loader` that you are concerned about?

Reply:
Seems like raw-loader has been deprecated for webpack v5

> DEPREACTED for v5: please consider migrating to [asset modules](https://v4.webpack.js.org/guides/asset-modules/).

Reply:
also seems like we can cleanup some remaining loaders from vite

Reply:
I moved to using asset modules in webpack5 and it works well. The only thing that the svg-inline-loader was doing for us (at least as configured) was stripping the `width` and `height` properties in the `svg` tag out for us, but I don't think it makes a difference in the rendered output in browsers.

Reply:
Ok, we needed to move to `asset/inline` as the asset module setting. I just learned this too, but FYI `asset/inline` makes webpack generate a `data` URI with base64 encoding so it can be inlined as a string to `src=""`. In this case, the SVGs are included as `img src={Icon}` so simply inlining the raw SVG tag fails. In gen2, it looks like we emit all the asset files and link them as relative URIs but since we have all SVG images in gen3 it probably makes sense to inline everything. This does add to the bundle size a bit so we should pay attention here and consider changing this in the future.